File: blepro/src/main/java/com/lepu/blepro/ext/lpbp2w/FileListCrc.java
package com.lepu.blepro.ext.lpbp2w;

public class FileListCrc {
    private int fileType;
    private int crc;

    public int getFileType() {
        return fileType;
    }

    public void setFileType(int fileType) {
        this.fileType = fileType;
    }

    public int getCrc() {
        return crc;
    }

    public void setCrc(int crc) {
        this.crc = crc;
    }

    @Override
    public String toString() {
        return "FileListCrc{" +
                "fileType=" + fileType +
                ", crc=" + crc +
                '}';
    }
}

File: blepro/src/main/java/com/lepu/blepro/ble/data/FileListCrc.kt
package com.lepu.blepro.ble.data

import com.lepu.blepro.utils.toUInt

class FileListCrc {
    var fileType: Int  // 文件类型 LeBp2wBleCmd.FileType.ECG_TYPE, BP_TYPE, USER_TYPE,
    var crc: Int       // 0表示文件不存在

    constructor(fileType: Int, crc: ByteArray) {
        this.fileType = fileType
        this.crc = toUInt(crc)
    }

    override fun toString(): String {
        return """
            fileType : $fileType
            crc : $crc
        """.trimIndent()
    }

}